components/addForm.js

```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components';
+
+export default function AddCharacterForm() {
+  return (
+    <StyledContainer>
+      <StyledInputField
+        type="text"
+        maxLength={20}
+        placeholder="Name"
+      ></StyledInputField>
+      <StyledTextarea
+        type="text"
+        maxLength={255}
+        placeholder="Enter here your information...
+        Attributes, Skills, Characteristics, Items, Character-Story, etc."
+      ></StyledTextarea>
+      <StyledButtonContainer>
+        <button>Save</button>
+        <button>Reset Form</button>
+      </StyledButtonContainer>
+    </StyledContainer>
+  );
+}
+
+const StyledContainer = styled.div`
+  display: grid;
+  gap: 5px;
+  margin: 10px;
+
+  div {
+    margin: 10px;
+  }
```

Comment:
If you style these divs in their own styled component then you don't need this nested styling. This makes the code less understandable since one html element is styled at two different places.